ramenpos/src/main/java/com/assamblypayments/ramenpos/Enums.java

```diff
@@ -0,0 +1,37 @@
+package com.assamblypayments.ramenpos;
+
+public class Enums {
```

Comment:
This class contains no enums, why is it called `Enums`?

Reply:
done